lib/recognizer/accounts.ex

```diff
@@ -162,9 +162,9 @@ defmodule Recognizer.Accounts do
       {:error, %Ecto.Changeset{}}
 
   """
-  def register_user(attrs) do
+  def register_user(attrs, opts \\ []) do
```

Comment:
Sweet, this is pretty much exactly what I was thinking. I'd also pass the options down to `maybe_notify_new_user/1` so we can disable that too.

Reply:
I'd think we still want the welcome email to send. I will confirm with sales that is what they expect.

Reply:
I think it's pretty weird that we would send them an email and immediately require them to reset their password to access it. I'd at least change the wording to it

Reply:
I don't think they'll be immediately doing anything with their account, which is why we're creating accounts for them. The welcome email is the first opportunity Marketing has to engage the customer.